presto-docs/src/main/sphinx/release/release-0.157.1-t.rst

```diff
@@ -0,0 +1,168 @@
+=================
+Release 0.157.1-t
+=================
+
+Presto 0.157.1-t is equivalent to Presto release 0.157.1, with some additional features and patches.
+
+Security
+--------
+* LDAP Authentication
+
+  
+SQL
+---
+* Support correlated scalar aggregation subqueries
+* Support correlated scalar subqueries
+* Additional prepared statement syntax: DESCRIBE INPUT and DESCRIBE OUTPUT
+
+
+Data Types
+----------
+* Additional Varchar(x) and Char(x) function implementations
```

Comment:
How about "specialization for functions" or "Native functions for varchar(x) and char(x) types"? I believe before those patches we also supported varchar(x) and char(x) though through coercion.